feature/bgp/policybase/otg_tests/aspath_and_community_test/aspath_and_community_test.go

```
@@ -144,13 +153,67 @@ func configureImportBGPPolicy(t *testing.T, dut *ondatra.DUTDevice, ipv4 string,
 	gnmi.Replace(t, dut, pathV4.Config(), policyV4)
 }
 
+// configureAsPolicy is used to configure vendor specific config statement.
+func configureAsPolicy(t *testing.T, dut *ondatra.DUTDevice) {
+	t.Helper()
+	var config string
+	gnmiClient := dut.RawAPIs().GNMI(t)
+
+	switch dut.Vendor() {
+	case ondatra.JUNIPER:
+		config = juniperCLI()
```

Lets not use the CLI to push the configuration to the device. if this OC path is not supported let the test FAIL.
Do we have any other option like configuring using native path?